<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupMain</class>
 <widget class="QWidget" name="SetupMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iutgram</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="startPage">
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="source.qrc">:/src/img/tel-banner11.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>310</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>7</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color : black;
font-weight:bold;
font-size: 20px;
text-align:center;
</string>
     </property>
     <property name="text">
      <string>IUTGRAM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start_signup">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>500</y>
       <width>331</width>
       <height>54</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color:#229ED9;
color:white;
font-size:18px;
font-weight:bold;
padding: 15px 0;
</string>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>350</y>
       <width>280</width>
       <height>42</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color : gray;
font-size:16px;</string>
     </property>
     <property name="text">
      <string>Welcome to official iutgram desktop app
please sinup or login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start_login">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>420</y>
       <width>331</width>
       <height>54</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color:#229ED9;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginPage">
    <widget class="QWidget" name="loginForm" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>571</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginForm{background-color:white;
border-radius:10px;
}</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font-weight:bold;
font-size:18px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:14px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Please Enter your username and password
to login to your account</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_login_username">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>451</width>
        <height>24</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#input_login_username
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_login_username:focus
{
border-bottom:1px solid #40ABE3;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>passsword</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_login_password">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>451</width>
        <height>24</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#input_login_password
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_login_password:focus
{
border-bottom:1px solid #40ABE3;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #5683A4;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login_login">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>490</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color:#5683A4;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login_cancel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>490</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color:#519BC7;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <zorder>widget</zorder>
    <zorder>loginForm</zorder>
    <zorder>btn_login_login</zorder>
    <zorder>btn_login_cancel</zorder>
   </widget>
   <widget class="QWidget" name="signupPage">
    <widget class="QWidget" name="signupForm" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>571</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#signupForm{
background-color:white;
border-radius:10px;
}</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font-weight:bold;
font-size:18px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>SignUp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:14px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Please Enter your personal information
to create an account</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signup_password">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>451</width>
        <height>24</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#input_signup_password
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_signup_password:focus
{
border-bottom:1px solid #40ABE3;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>passsword</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signup_username">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#input_signup_username
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_signup_username:focus
{
border-bottom:1px solid #40ABE3;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>firstname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>lastname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_firstname">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>451</width>
        <height>24</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#input_firstname
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_firstname:focus
{
border-bottom:1px solid #40ABE3;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_lastname">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>451</width>
        <height>24</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#input_lastname
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_lastname:focus
{
border-bottom:1px solid #40ABE3;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #5683A4;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_signup_signup">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>520</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color:#5683A4;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_signup_cancel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>520</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color:#519BC7;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <zorder>widget_4</zorder>
    <zorder>signupForm</zorder>
    <zorder>btn_signup_signup</zorder>
    <zorder>btn_signup_cancel</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
